Modules/_cursesmodule.c

```diff
@@ -4681,25 +4699,25 @@ static PyMethodDef PyCurses_methods[] = {
 
 static struct PyModuleDef _cursesmodule = {
     PyModuleDef_HEAD_INIT,
-    "_curses",
-    NULL,
-    -1,
-    PyCurses_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
+    .m_name = "_curses",
+    .m_doc = NULL,
+    .m_size = -1,
+    .m_nethods = PyCurses_methods,
+    .m_slots = NULL,
```

Comment:
Remove them?

Reply:
@shihai1991 I will re-add in the next PR (to change to multi-phase init) so I thought I might as well add them now and keep as NULL.